scheduler/test/cook/test/mesos/util.clj

```diff
@@ -562,4 +563,82 @@
     (is (= 12 (util/cache-lookup! cache-store "A" 12)))
     (is (= {"A" 12, "C" 31} (.cache @cache-store)))))
 
+
+(deftest test-retry-job
+  (let [uri "datomic:mem://test-retry-job"
+        conn (restore-fresh-database! uri)]
+    (testing "increment retries on running job"
+      (let [[job _] (create-dummy-job-with-instances conn :job-state :job.state/running
+                                                     :instances [{:instance-status :instance.status/running}])
```

Comment:
Might be nice to explicitly set the `:retry-count` here to make it more clear

Reply:
Done